### ISSUE TITLE
Support single-quoted strings as alias for double-quoted strings in Rhai language parser

### ISSUE THREAD

**Issue by shah** [rhaiscript/rhai#912]
Per our discussion in #911 there is a desire to treat single-quotes like ' the same as double-quotes like " (the same way JavaScript handles strings flexibly). This feature would allow non-technical users of scripting language to have three quotation modes - non-interpolated via " and ' and interpolated via back-tick.

**Comment by schungx** [comment]
Yes, let me ponder whether this would be simple to add...  probably so as the literal string parsing function is quite flexible.

**Comment by schungx** [comment]
There will still be differences regarding how SQL treat strings and how C-style languages treat strings, e.g. wrt escaping etc.

**Comment by schungx** [comment]
Closing this for now as not obvious that this will be a significant improvement.
